add additional information on Wildcard Domains (#223)
railwayapp/docs: src/docs/deploy/exposing-your-app.md

layout="responsive"
width={1338} height={808} quality={80} />

**NOTE:** For wildcard domains, see the section below.

**NOTE:** Changes to DNS settings may take up to 72 hours to propagate
worldwide.

**NOTE:** Freenom is not allowed, and not supported.

## Wildcard Domains
There are a few important things to know when using Wildcard Domains.
1. Ensure that the CNAME record for authorize.railwaydns.net is not proxied by your provider (eg: Cloudflare). This is required for the verification process to work.

> 2. Sibling domains (eg: thing1.yourdomain.com) can be proxied, even if you're using a wildcard (eg: *.yourdomain.com).

Reviewer: I'm not sure what this means

> 3. The wildcard can be used for level of subdomain (eg: *.yourdomain.com) or deeper (eg: *.subdomain.yourdomain.com).

Reviewer: ```suggestion
3. Wildcards can be used for any subdomain level (eg: `*.yourdomain.com` or  `*.subdomain.yourdomain.com`).
```
